docs/docs/general/07-Troubleshooting/askui-ui-controller-starting-problems.md

```diff
@@ -0,0 +1,27 @@
+---
+custom_edit_url: null
+---
+
+# askui UI Controller Starting Problems
+
+## Wayland
+
+At the moment we don't support the windowing system Wayland for any linux distribution. 
+As solution you have to use Xorg instead of Wayland. You can look on [this page](https://www.maketecheasier.com/switch-xorg-wayland-ubuntu1710/) to see how to Switch to Xorg from Wayland.
```

Comment:
```suggestion
As a solution, you can switch to Xorg instead of Wayland. [ how to switch to Xorg](https://www.maketecheasier.com/switch-xorg-wayland-ubuntu1710/) to see how to Switch to Xorg from Wayland.
```